WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/AddOrderCoordinator.swift

```diff
@@ -86,3 +107,34 @@ private extension AddOrderCoordinator {
         ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.orderAddNew())
     }
 }
+
+final class NewSimplePaymentsNoticeViewController: UIViewController {
```

Comment:
Nice naming! I would add something to let it know that Simple Payments are now in a different app place, otherwise, it seems like we are using a new simple payments feature. Something like `NewSimplePaymentsLocationNoticeViewController` ?

Reply:
Done! 22da3170258ac0e5eebdcbeff75e1c98a5098fc5